89.md

```diff
@@ -0,0 +1,81 @@
+NIP-89
+======
+
+Bitcoin Transaction Package Relay
+-------------------------
+
+`draft` `optional` `author:benthecarman`
+
+A special event with kind `28333`, that contains encoded bitcoin transactions.
+The bitcoin transactions are encoded as a hex string, and are contained in a `transactions` tag.
+
+The network magic is contained in the `tags` field. This is to identify the network that the transactions are for.
+The common ones are:
+
+- mainnet: f9beb4d9
+- testnet3: 0b110907
+- signet (default): 0a03cf40
+- regtest: fabfb5da
```

Comment:
> Might be better to use network magic so we can delineate between signets as well.

-- https://github.com/nostr-protocol/nips/pull/476#issuecomment-1525038776

Reply:
Right. But I suppose those alternative signets will also get their own name?

Reply:
I think this is better, this way we don't get any overlap, this could even be used for things like litecoin without needing new specifications.

I do agree just writing "mainnet" is more clear but I figured just enumerating them here would make it easy enough. Also, most bitcoin libraries will have these values available anyways.